controllers/ProfileController.js

```diff
@@ -69,6 +70,62 @@ class ProfileController {
       user: userData
     });
   }
+
+  /**
+   * @author Manzi
+   * @param {*} req
+   * @param {*} res
+   * @param {*} next
+   * @returns {*} Users object
+   */
+  static async getProfiles(req, res) {
+    const { currentUser } = req;
+    let include;
+    if (currentUser) {
+      include = {
+        model: Follow,
+        as: 'userFollower',
+        where: { follower: currentUser.id },
+        required: false,
+        attributes: ['followee']
+      };
+    }
+    let users = await User.findAll({
+      where: {
+        userType: 'user',
+        confirmed: { [Op.ne]: 'pending' },
+        status: { [Op.ne]: 'blocked' }
+      },
+      include,
+      attributes: ['firstName', 'lastName', 'image', 'bio']
+    });
+    users = users.map(data => {
+      const user = { ...data.get() };
+      user.followed = user.userFollower && user.userFollower.length > 0;
+      delete user.userFollower;
+      return user;
+    });
+    return res.status(200).json({ status: 200, profiles: users });
```

Comment:
@NkFab You need to paginate the users' list as the list might be too long

Reply:
Ok let me do that